service-api/module/Application/src/Controller/YotiController.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Application\Controller;
+
+use Application\Yoti\YotiServiceInterface;
+use Laminas\Mvc\Controller\AbstractActionController;
+use Laminas\Http\Response;
+use Laminas\View\Model\JsonModel;
+
+/**
+ * @psalm-suppress PropertyNotSetInConstructor
+ * Needed here due to false positive from Laminas’s uninitialised properties
+ */
+class YotiController extends AbstractActionController
+{
+    public function __construct(
+        private readonly YotiServiceInterface $yotiService
+    ) {
+    }
+
+    public function findPostOfficeAction(string $postCode): JsonModel
+    {
+        $branches = [];
+        $branches["locations"] = $this->yotiService->postOfficeBranch($postCode);
```

Comment:
This isn't covered by tests, so hard to confirm, but I don't think this matches the structure expected in [ID-137](https://opgtransform.atlassian.net/browse/ID-137).

Reply:
Created this purely as a stub/placeholder before ID-137 existed, plan is to tackle this next